src/documentation/pages/components/alert/_properties.vue

```diff
@@ -0,0 +1,169 @@
+<template lang="pug">
+//- TODO: uncomment once stencilDoc is implemented
```

Comment:
Let's remove the hardcoded properties. We will implement stencilDoc alternative in a separate PR.